api/WEBGL_depth_texture.json

```diff
@@ -32,13 +32,20 @@
             },
             {
               "prefix": "MOZ_",
-              "version_added": true,
+              "version_added": "17",
+              "version_removed": "58"
```

Comment:
I can't find evidence of this being removed in Firefox 58 in https://bugzilla.mozilla.org/show_bug.cgi?id=738866, can you point it out?

Reply:
I had confirmed this via manual testing:

![Screenshot from 2021-09-16 12-46-24](https://user-images.githubusercontent.com/5179191/133676282-9ae9cea3-6bd4-4721-85c0-866ff6f2e244.png)

Reply:
I see, thanks! Noting that in the description would be great, I'll edit it in.